README

```diff
@@ -4,15 +4,11 @@
 ------------------------------------------------------------
 
 This README is intended to provide quick and to-the-point documentation for
-technical users intending to compile parts of Guacamole themselves.
+technical users intending to compile parts of Apache Guacamole themselves.
 
-Source archives are available from the files section of the main project page:
```

Comment:
This should probably still be here, but instead refer to the "downloads" section. It is still valid, though definitely should not point to SourceForge.